experiments/exp934_hq_vs_pt.py

```diff
@@ -89,7 +89,7 @@
     tokenizer=llama3_tokenizer,
 ).with_output_path("tokenized/stackexchange-621b94")
 
-full_mix_components = {
+pt_vs_hq_components = {
```

Comment:
don't understand what pt_vs_hq means?

Reply:
I mean, I get pt = pretraining and hq = high quality, but why vs?  what about pretraining_high_quality_components?

Reply:
it's the name of the experiment file. pretraining vs high quality

Reply:
My name originally - I am bad at the shorthand names! It's "v.s." because it ran an ablation comparing just cooling down on Pretraining v.s. just cooling down on HQ components.